Poker.FE/Assets/MainMenu/Scripts/MainMenu.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/// <summary>
+/// Attached to Main Camera
+/// </summary>
+public class MainMenu : MonoBehaviour {
+    public Texture backgroundTexture;
+
+    private void OnGUI()
```

Comment:
nice background! 👍 
![image](https://cloud.githubusercontent.com/assets/9889268/26220027/bc28788c-3c19-11e7-85c9-cbe558c07934.png)
i'm trying to find the line you load it